feconf.py

```diff
@@ -335,14 +337,14 @@
         'content': {},
         'default_outcome': {}
     }
-}
+} # type: Dict[Text, Dict[Text, Dict[Text, Text]]]
 # Default written_translations dict for a default state template.
 DEFAULT_WRITTEN_TRANSLATIONS = {
     'translations_mapping': {
         'content': {},
         'default_outcome': {}
     }
-}
+} # type: Dict[Text, Dict[Text, Dict[Text, Text]]]
```

Comment:
Ah yes! I get you. I was also confused and wanted to ask do we want to do this? @seanlip suggested me to use [TypedDict](https://docs.python.org/3/library/typing.html#typing.TypedDict). But the documentation mentions 'New in version 3.8'. So I don't think we can use it here. We will be migrating to py3.7. 
And one more point, type annotations in the form of TypedDict will make our code messier. In typescript, the type is defined just after the variable but in python, this will make our code look clumsy.
Example:
```
from typing import TypedDict

Point2D = TypedDict('Point2D', {
  'x': int, 
  'y': int, 
  'label': str
})

a = {'x': 1, 'y': 2, 'label': 'good'} # type: Point2D

```
What do you think? @sajalasati @DubeySandeep

Reply:
The example looks fine and simple to me. (Maybe the name should have a `Type` prefix?)

Reply:
Yes Type prefix can be added. But still it is new in version 3.8, can we use this in py3.7?
And my point still stays, I think this TypedDict will make our code clumpsier.

Reply:
I think what @hardikkat24 is trying to suggest here is is that such a case might not be frequent (i.e. outside of constants file), and to decide whether to implement it codebase-wide or not, maybe we can discuss this in detail separately (what to do and the plan that we should have) - since anyways it can be implemented only after we start using python3.8+. What do you suggest?